portality/migrate/seal/seal.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Actually, sorry for q after merge but how does this add the new seal field? I understand this just specifies which types to apply the changes to, but where are the changes actually defined? Or is it a purely new field created upon saving an object (in which case why use a migration)?